spec/integration/jara_spec.rb

```diff
@@ -46,20 +38,35 @@ def setup_git(project_dir)
     isolated_run(project_dir,
       'git init --bare ../repo.git',
       'git init',
-      'git add . && git commit -m "Fist!"',
+      'git add .',
+      'git commit -m "First!"',
       'git remote add origin ../repo.git',
       'git push -u origin master'
     )
   end
 
+  def bundle_env(dir)
+    {
+      'BUNDLE_GEMFILE' => File.join(dir, 'Gemfile'),
+      'BUNDLE_PATH' => Dir[File.join(VENDOR_PATH, File.basename(dir), '*', '*')].sort.last,
+    }.map { |k, v| "#{k}=#{v}" }.join(' ')
```

Comment:
I don't think this will work if `dir` contains characters that need to be escaped, e.g. if the user clones Jara into a directory that has spaces in its name.

You could use something like [Shellwords](http://ruby-doc.org/stdlib-1.9.3/libdoc/shellwords/rdoc/Shellwords.html) to escape `v` or you could use 
```ruby
IO.popen([env, cmd, *args], 'r', &:read)
```
to run the commands with the proper `env`.